tests/validator/validator.cpp

```diff
@@ -51,10 +51,13 @@ TEST(Validator, unnamedModel)
     const std::vector<std::string> expectedSpecificationHeadings = {
         "2.1.1",
     };
+    const std::vector<std::string> expectedUrls = {
+        "https://cellml-specification.readthedocs.io/en/latest/reference/formal_and_informative/specB01.html?issue=MODEL_NAME",
+    };
     libcellml::ValidatorPtr validator = libcellml::Validator::create();
```

Comment:
To be consistent we should have an empty line here.

Reply:
Done in commit a06be2cd.